electricpy/__init__.py

```diff
@@ -6573,9 +6573,277 @@ def wireresistance(length=None,diameter=None,rho=16.8*10**-9,R=None):
         A = rho*length/R
         return _np.sqrt(4*A/pi)
         
+def ic_555(mode,R=None,C=None,f=None,t_high=None,t_low=None):
```

Comment:
I've started to move away from `f` in favor of `freq` let's see if we can change that out here in this function as well.

Actually... this brings up a question; it doesn't look like the argument `f` is actually used in the body of the code. Is this intentional? Is it intended to be an "override" option?

Reply:
ic_555(mode,R=None,C=None,freq=None,t_high=None,t_low=None): not its not intended to overide